documentation/usage/settings.md

```diff
@@ -2,8 +2,101 @@
 sidebar_position: 1
 ---
 
+import CaptionedImage from "/src/components/CaptionedImage";
+
 # Game Settings
 
+In addition to the settings available in the original Jak games, OpenGOAL provides some additional settings that will allow you to tweak and improve the games in new ways. This page will highlight some of these new settings and provide examples for them, but there are still more to find in game. Once you've had a look through this page, [try changing them yourself in game](/docs/usage/installation)!
+
+## Subtitles (and languages)
+
+**Found in Options > Game Options**
+
+One major new addition to OpenGOAL that was not present in the original game are the subtitles. These were not part of the original game, but instead are a community effort to transcribe and translate the game into subtitles for different languages, making it easier for more people to enjoy.
+
+Additionally all of the original available audio and text options for the game are available as settings, regardless of what version is on your disc!
+
+This list may expand as more languages are added, but for now these are the available options:
+
+Audio: ENGLISH, FRANÇAIS, DEUTSCH, ESPAÑOL, ITALIANO, にほんご
+
+Subtitles: ENGLISH, DEUTSCH, ESPAÑOL, PORTUGUÊS (BRASIL)
+
+Text language: : ENGLISH, ENGLISH (UK), FRANÇAIS, DEUTSCH, ESPAÑOL, ITALIANO, にほんご
+
+If you're interested in adding new subtitles to the game, or editing existing ones, take a look [at the subtitle translations page](/docs/contributing/subtitle-translations).
+
+![Subtitle and language settings](/docs/settings/settings_subtitles.png)
+![An example of subtitles in game](/docs/settings/subtitles_example.png)
+
+## Cutscene skips
+
+**Found in Options > Game Options > Miscellaneous**
+
+Speedrunners, rejoice! This setting enables you to skip most of the cutscenes in the game by pressing Triangle (or the equivalent button) while they're playing. There are many smaller cutscenes that are not technically cutscenes in the code, and these can't be skipped (eg the Klaww battle start scene, or the Mountain pass start scene), but this option still saves a significant amount of time.
```

Comment:
thinking out loud again, having a component for controller icons would also be nice

Reply:
Yeah I've been meaning to look into how the key and controller binding works in OpenGOAL too, will need that to fill out the bindings section and might be able to make a nice controls display as part of it. Not sure how much work that would end up being but I'll see if I can find a simple option with the bindings, and fix it up here if so